lti_consumer/lti_consumer.py

```diff
@@ -717,7 +714,7 @@ def result_service_handler(self, request, suffix=''):
 
         user = self.runtime.get_real_user(anon_id)
         if not user:  # that means we can't save to database, as we do not have real user id.
-            msg = "[LTI]: Real user not found against anon_id: {}".format(anon_id)
+            msg = self.ugettext("[LTI]: Real user not found against anon_id: {}").format(anon_id)
```

Comment:
Looks like this message only goes to logs. Should we be marking this for translation?